fili-core/src/test/groovy/com/yahoo/bard/webservice/web/apirequest/ApiRequestImplSpec.groovy

```diff
@@ -28,6 +26,7 @@ class ApiRequestImplSpec extends Specification {
         String tableName = "tableName"
         Granularity granularity = Mock(Granularity)
         LogicalTableDictionary logicalTableDictionary = Mock(LogicalTableDictionary)
+        logicalTableDictionary.getNames() >> [tableName]
```

Comment:
Need to do a tableName scan before `LogicalTable` lookup to make sure tableName exist.